homeassistant/components/harmony/strings.json

```
@@ -41,5 +41,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_switches": {
+      "title": "The Logitech Harmony switch platform is being removed",
+      "description": "Using the switch platform to change the current activity is now deprecated and will be removed in a future version of Home Assistant.\n\nPlease adjust any automations or scripts that you may have to use the select entity that is already available to fix this issue."
```

```suggestion
      "description": "The switch entities have been replaced with a select entity. Using the switch platform to change the current activity is now deprecated and will be removed in a future version of Home Assistant.\n\nPlease adjust any automations or scripts that you may have to use the select entity already available to fix this issue."
```

We should tell them why so they don't ask 👍

---

Just saw this after updating the PR.